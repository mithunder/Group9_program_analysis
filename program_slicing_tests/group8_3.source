module group81 :
x := 0;
y := 7;
z := 2;
i := 0;
j := y;
k := z;
if 
true -> z := z * y + z * x  []
true -> z := x + x + x - x []
true ->
@programslicing="true"
	x := x + y + z * z
fi;
do
   y < 100 -> y := y + y
od;
write y
end
