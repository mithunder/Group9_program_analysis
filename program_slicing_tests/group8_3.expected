module group81 :
x := 0;
y := 7;
z := 2;
if 
true -> skip []
true -> skip []
true -> x := x + y + z * z
fi
end
